linera-execution/src/wasm/mod.rs

```diff
@@ -268,40 +246,10 @@ const _: () = {
     }
 };
 
-/// Errors that can occur when executing a user application in a WebAssembly module.
-#[cfg(any(with_wasmer, with_wasmtime))]
-#[derive(Debug, Error)]
-pub enum WasmExecutionError {
```

Comment:
I feel that this one should stay as it is, and you would just create a new type of Error for the EVM.

Reply:
It's fair normal for each module to have its Error type.

Reply:
Ok, let's kill the PR.